src/main/java/com/buschmais/jqassistant/scm/maven/ScanMojo.java

```diff
@@ -94,8 +95,16 @@ public void execute(MavenProject mavenProject, Store store) throws MojoExecution
         store.beginTransaction();
         try {
             scanner.scan(mavenProject, mavenProject.getFile().getAbsolutePath(), MavenScope.PROJECT);
-        } finally {
-            store.commitTransaction();
+        } catch (Throwable e) {
```

Comment:
Throwable should be replaced by RuntimeException as we won't suppress things like OOMs, etc.

Reply:
Probably. I tried to be as close as possible to the previous code, which did commit the transaction even in case of an OOM. But if you think it was a mistake even in the previous code, I'll change it.